alot/commands/envelope.py

```diff
@@ -168,6 +168,33 @@ def __init__(self, mail=None, envelope=None, **kwargs):
         self.envelope = envelope
         self.envelope_buffer = None
 
+    def _check_recipient_keys(self):
+        # get all recipient addresses
+        recipients = set()
+        for header in ['To', 'Cc', 'Bcc']:
```

Comment:
in other places, we chose not to include BCCs when encrypting. Let's be consistent.